### PR TITLE
Remove soft-failure from test

### PR DESCRIPTION
**Pull request by eliroca**
Remove soft-failure from test for SLE 15

- The bug is already fixed: https://build.suse.de/request/show/184097

- Related ticket: https://bugzilla.suse.com/show_bug.cgi?id=1123173

### CODE REVIEW

**Comment by okurz** [comment]
Well, the bug status is "New" so not yet really fixed, right?

15.01.2020 19:28:28 Elisei Roca <notifications@github.com>:

**Comment by eliroca** [comment]
It was fixed about a year ago with this request: https://build.suse.de/request/show/184097

**Comment by okurz** [comment]
Yes, I can see that but the bug is *still* in status "New" so not reflecting that you consider it "Fixed". And we should keep soft failure references in the code until the bug is VERIFIED and at best ensured to not be present in any tested product version.

**Comment by eliroca** [comment]
The error occurred because zypper-docker was not updated to 2.0.0 on SLE 15. That's been fixed.

I ran the steps to reproduce this on SLES 15 SP1 and openSUSE Leap 15.1 VMs and can confirm that the bug is gone. I also ran the test successfully on my local openQA deployment by cloning the job.
I will edit the commit in order to keep the soft-failure reference.

If there is anything else I can do, please let me know.

File: tests/console/zypper_docker.pm [hunk 1]
```diff
@@ -48,11 +47,8 @@ sub run {
     assert_script_run("zypper-docker list-updates-container tmp_container", timeout => 600);
     assert_script_run("zypper-docker list-patches-container tmp_container", timeout => 600);
     # apply all the updates to a new_image
-    if (is_sle('>=15')) {
-        record_soft_failure 'bsc#1123173';
-    } else {
-        assert_script_run("zypper-docker update --auto-agree-with-licenses $testing_image new_image", timeout => 900);
-    }
+    # if (is_sle('>=15')) record_soft_failure 'bsc#1123173'
```
Comment (foursixnine) [inline tests/console/zypper_docker.pm]: @eliroca Mind removing this line?. It isn't needed anymore :)
```suggestion
```

Comment (eliroca) [reply tests/console/zypper_docker.pm]: I could remove it, but @okurz made a good point in recommending we should keep the soft failure reference for a while.

Comment (foursixnine) [reply tests/console/zypper_docker.pm]: @okurz is the first person to complain when there's disabled code :). Leave a comment instead if you prefer.

Comment (okurz) [reply tests/console/zypper_docker.pm]: foursixnine is right :)

Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit. Having a comment in plain english is better.
